lib/Capnp/Message.hs

```diff
@@ -36,6 +36,10 @@ module Capnp.Message (
     , ConstMsg
     , empty
     , singleSegment
+    , constSegs
+    , constMsgFromSegments
+    , constSegToVec
+    , constSegFromVec
```

Comment:
It looks like these were developed against an older release? `ConstMsg` itself isn't exposed anymore on master, so these are bitrotten (there's now `Message` and `Segment` types with a type parameter of kind `Mutability`). I'd be ok with merging additions like these, but they need to be updated for the changes on master, of course.